pkg/controlplane/controller.go

```diff
@@ -79,8 +79,7 @@ type Controller struct {
 	ServicePort               int
 	PublicServicePort         int
 	KubernetesServiceNodePort int
-
-	runner *async.Runner
```

Comment:
nit, avoid this noop changes , I assume runner is separate because is private

Reply:
done